src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -27,6 +27,76 @@ public sealed partial class ApplicationIdentity : System.Runtime.Serialization.I
     }
 }
 
+namespace System.Data.Common
+{
+    public abstract partial class DBDataPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        [Obsolete]
```

Comment:
(Though one has to wonder if `[Obsolete(true)]` should be just deleted here, since someone would be recompiling anyways...   @weshaggard are we deleting Obsolete(true) during ref-ports?)